client/my-sites/pages/page-list.jsx

```diff
@@ -220,6 +181,43 @@ class Pages extends Component {
 		);
 	}
 
+	showBlogPostsPage() {
+		const { site, homepageType, isFSEActive } = this.props;
+		const { search, status } = this.props.query;
+
+		return (
+			( ! config.isEnabled( 'unified-pages/virtual-home-page' ) ||
+				/** Blog posts page is for themes that don't support FSE */
+				! isFSEActive ) &&
+			site &&
+			homepageType === 'posts' &&
+			/** Under the "Published" tab */
+			status === 'publish,private' &&
+			/** Without any search term */
+			! search
+		);
+	}
+
+	/**
+	 * Show the virtual homepage
+	 */
+	showVirtualHomepage() {
+		const { site, homepageType, blockEditorSettings, isFSEActive, translate } = this.props;
+		const { search, status } = this.props.query;
+
+		return (
+			config.isEnabled( 'unified-pages/virtual-home-page' ) &&
+			/** Virtual homepage is for themes that support FSE */
+			isFSEActive &&
+			site &&
+			homepageType === 'posts' &&
```

Comment:
Not sure if it is in the scope of this PR, but please see: https://github.com/Automattic/wp-calypso/issues/69171#issuecomment-1283748835

Reply:
replied